shell.nix

```diff
@@ -45,51 +30,60 @@ let
   ]);
 
   baseEditorScripts = [
+    (pkgs.writeScriptBin "install-utopia-api" ''
+      #!/usr/bin/env bash
+      set -e
+      cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)/utopia-api
+      pnpm install
+      pnpm run build
+    '')
```

Comment:
This was added to replace the script previously living in the editor's `package.json`, which was causing build issues